raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -763,6 +755,11 @@ contract TokenNetwork is Utils {
             returned_tokens
         );
 
+        // After the channel is settled the storage is cleared, therefor the
+        // value will be Gone and not Settled. The value Settled is used for
```

Comment:
`Gone` -> `NonExistent`

Reply:
TIL: I thought that non-existent was a typo, and that only non-existant is correct but I was wrong as they are both acceptable spellings.